releases/release-5.2.4.md

```diff
@@ -0,0 +1,186 @@
+    - (dup: release-5.3.1.md > Bug fixes> TiKV)- Fix the issue that the latency of Resolved TS increases after TiKV stops operating [#11351](https://github.com/tikv/tikv/issues/11351)
+    - (dup: release-5.3.1.md > Bug fixes> TiKV)- Fix a panic issue that occurs when Region merge, ConfChange, and Snapshot happen at the same time in extreme conditions [#11475](https://github.com/tikv/tikv/issues/11475)
+    - (dup: release-5.3.1.md > Bug fixes> TiKV)- Fix a bug that tikv-ctl cannot return the correct Region-related information [#11393](https://github.com/tikv/tikv/issues/11393)
+    - (dup: release-5.0.6.md > Bug fixes> TiKV)- Fix the issue of negative sign when the decimal divide result is zero [#29586](https://github.com/pingcap/tidb/issues/29586)
+    - (dup: release-5.4.0.md > Bug fixes> TiKV)- Fix the issue that retrying prewrite requests in the pessimistic transaction mode might cause the risk of data inconsistency in rare cases [#11187](https://github.com/tikv/tikv/issues/11187)
+    - (dup: release-5.3.0.md > Bug Fixes> TiKV)- Fix a memory leak caused by monitoring data of statistics threads [#11195](https://github.com/tikv/tikv/issues/11195)
+    - (dup: release-5.3.1.md > Bug fixes> TiKV)- Fix the issue that the average latency of the by-instance gRPC requests is inaccurate in TiKV metrics [#11299](https://github.com/tikv/tikv/issues/11299)
+    - (dup: release-6.0.0-dmr.md > Bug fixes> TiKV)- Fix the panic issue caused by deleting snapshot files when the peer status is `Applying` [#11746](https://github.com/tikv/tikv/issues/11746)
+    - (dup: release-6.0.0-dmr.md > Bug fixes> TiKV)- Fix a bug that TiKV cannot delete a range of data (which means the internal command `unsafe_destroy_range` is executed) when the GC worker is busy [#11903](https://github.com/tikv/tikv/issues/11903)
+    - (dup: release-6.0.0-dmr.md > Bug fixes> TiKV)- Fix the issue that deleting an uninitialized replica might cause an old replica to be recreated [#10533](https://github.com/tikv/tikv/issues/10533)
+    - (dup: release-5.3.1.md > Bug fixes> TiKV)- Fix the issue that TiKV cannot detect the memory lock when TiKV performs a reverse table scan [#11440](https://github.com/tikv/tikv/issues/11440)
+    - (dup: release-5.3.1.md > Bug fixes> TiKV)- Fix the deadlock issue that happens occasionally when coroutines run too fast [#11549](https://github.com/tikv/tikv/issues/11549)
+    - (dup: release-6.0.0-dmr.md > Bug fixes> TiKV)- Fix the issue that destroying a peer might cause high latency [#10210](https://github.com/tikv/tikv/issues/10210)
+    - Fix TiKV panic and peer unexpectedly destroyed due to fake merge target [#12232](https://github.com/tikv/tikv/issues/12232)
+    - Fix panic when target peer is replaced with a destroyed uninitialized peer during merge [#12048](https://github.com/tikv/tikv/issues/12048)
+    - Fix panic when applying snapshot is aborted [#11618](https://github.com/tikv/tikv/issues/11618)
```

Comment:
```suggestion
    - Fix the issue that TiKV panics and destroys peers unexpectedly because the target Region to be merged is faked [#12232](https://github.com/tikv/tikv/issues/12232)
    - Fix the TiKV panic issue that occurs when the target peer is  replaced with a destroyed uninitialized peer when merging a Region [#12048](https://github.com/tikv/tikv/issues/12048)
    - Fix the TiKV panic  issue that occurs when applying snapshot is aborted [#11618](https://github.com/tikv/tikv/issues/11618)
```